sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample10_MultiLabelClassify.md

```diff
@@ -34,54 +34,35 @@ List<string> batchedDocuments = new()
 // classify your documents, see https://aka.ms/azsdk/textanalytics/customfunctionalities.
 string projectName = "<projectName>";
 string deploymentName = "<deploymentName>";
-MultiLabelClassifyAction multiLabelClassifyAction = new(projectName, deploymentName);
-
-TextAnalyticsActions actions = new()
-{
-    MultiLabelClassifyActions = new List<MultiLabelClassifyAction>() { multiLabelClassifyAction }
-};
 
 // Perform the text analysis operation.
-AnalyzeActionsOperation operation = await client.StartAnalyzeActionsAsync(batchedDocuments, actions);
-await operation.WaitForCompletionAsync();
+ClassifyDocumentOperation operation = await client.MultiLabelClassifyAsync(WaitUntil.Completed, batchedDocuments, projectName, deploymentName);
```

Comment:
nitpick: should we simplify batchedDocuments to just documents? it feels batched is redundant given the plural.

Reply:
I agree the name of this variable could be better. I think I'll keep it unchanged here to avoid making this PR bigger and will make the change separately.